junit/schema/testdrive.xsd

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema
+	xmlns:xs="http://www.w3.org/2001/XMLSchema"
+	 elementFormDefault="qualified"
+	 attributeFormDefault="unqualified">
+	<xs:annotation>
+		<!-- modified: complement original documentation -->
+		<xs:documentation xml:lang="en">A schema for testdrive JUnit test results.
+
+testdrive emits test results which are not strictly compatible with the Windy
```

Comment:
Since we are modifying the schema it would be good to explicitly note in the time field descriptions that the time is NOT execution duration, but the duration covered by the data being analysed for the test.